### DISCUSSION TITLE
Is 'Stake my tokens' confusing?

### DISCUSSION THREAD

**Discussion by Patrick1904** [near/near-wallet#1166] [General]
https://github.com/near/near-wallet/projects/4#card-49484002

**Comment by heycorwin** [comment]
The point about balances here is valid and is being addressed elsewhere. The button copy thing is something I've only heard from this individual. I'm happy to change it to something else if this ends up being feedback that persists, but would like to know if this is something that we have heard from multiple individuals...

At the end of the day, lots of people are going to have different opinions about what copy they think is most clear. Personally I don't see how `Stake my tokens` could be any more clear, but it's the users' opinion that matter most. Let's continue to monitor this.
